### ISSUE TITLE
selenium webdriver promise error

### ISSUE THREAD

**Issue by sethkane** [felixzapata/gulp-axe-webdriver#9]
Any thoughts?

node v6.9.4
npm 3.10.10
gulp-axe-webdriver ^1.3.1

`/Users/sethk/Documents/workspace/seth-project/node_modules/selenium-webdriver/lib/promise.js:2517
        throw error;
        ^

Error: Timed out waiting for the WebDriver server at http://10.101.24.41:50516/
    at Error (native)
    at onError (/Users/sethk/Documents/workspace/seth-project/node_modules/selenium-webdriver/http/util.js:102:16)
    at process._tickCallback (internal/process/next_tick.js:103:7)
From: Task: WebDriver.createSession()
    at Function.createSession (/Users/sethk/Documents/workspace/seth-project/node_modules/selenium-webdriver/lib/webdriver.js:777:24)
    at Function.createSession (/Users/sethk/Documents/workspace/seth-project/node_modules/selenium-webdriver/phantomjs.js:220:55)
    at createDriver (/Users/sethk/Documents/workspace/seth-project/node_modules/selenium-webdriver/index.js:167:33)
    at Builder.build (/Users/sethk/Documents/workspace/seth-project/node_modules/selenium-webdriver/index.js:654:16)
    at module.exports (/Users/sethk/Documents/workspace/seth-project/node_modules/gulp-axe-webdriver/index.js:38:68)
    at Gulp.<anonymous> (/Users/sethk/Documents/workspace/seth-project/gulpfile.babel.js:327:16)
    at module.exports (/Users/sethk/Documents/workspace/seth-project/node_modules/orchestrator/lib/runTask.js:34:7)
    at Gulp.Orchestrator._runTask (/Users/sethk/Documents/workspace/seth-project/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/sethk/Documents/workspace/seth-project/node_modules/orchestrator/index.js:214:10)
    at /Users/sethk/Documents/workspace/seth-project/node_modules/orchestrator/index.js:279:18
    at finish (/Users/sethk/Documents/workspace/seth-project/node_modules/orchestrator/lib/runTask.js:21:8)
    at module.exports (/Users/sethk/Documents/workspace/seth-project/node_modules/orchestrator/lib/runTask.js:60:3)
    at Gulp.Orchestrator._runTask (/Users/sethk/Documents/workspace/seth-project/node_modules/orchestrator/index.js:273:3)
    at Gulp.Orchestrator._runStep (/Users/sethk/Documents/workspace/seth-project/node_modules/orchestrator/index.js:214:10)
    at Gulp.Orchestrator.start (/Users/sethk/Documents/workspace/seth-project/node_modules/orchestrator/index.js:134:8)
    at Gulp.<anonymous> (/Users/sethk/Documents/workspace/seth-project/node_modules/gulp/index.js:36:18)`

**Comment by felixzapata** [comment]
hi @sethkane , are you trying to analyze a local or a remote url?

**Comment by sethkane** [comment]
I am trying to analyze a full URL but on my local machine http://local.styleguide.com
I have that in my host file and it points to my VM on my machine.
I also have browser sync proxying that URL

Here is my config file that might put some context to it:

{
    "env":{
        "dev":"./clientname.com",
		"wwwroot":"/Volumes/wwwroot/clientname/Website"
    },
    "proxyUrl": ["http://local.styleguide.com"],
    "sitesToBuild": ["style-guide","wisconsin"],
    "a11y": {
        "urls":[
        	"http://local.styleguide.com/Service-Line-Detail/Heroes/Version1"
        ],
        "enabled": false,
        "include": "",
        "exclude": ".hljs"
    }
}

So basically localhost:9000 proxies local.styleguide.com via Browser sync.
I want to test a page on localhost.styleguide.com which is on my local VM

Make sense?

**Comment by felixzapata** [comment]
mmm does it work if you test the localhost?. In other project I tested urls based on virtual hosts and it worked. I mean, I added on my `vhosts` file the virtual urls.
